app/build.gradle

```diff
@@ -40,4 +41,8 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+
+    dependencies {
+        implementation 'com.airbnb.android:lottie:5.0.3'
```

Comment:
I think you can write just the implementation line and it will work. It doesn't need the 'dependencies' thing 👍